boards/tests/test_forms.py

```diff
@@ -0,0 +1,23 @@
+from django.urls import reverse
+from django.test import SimpleTestCase, TestCase
+from django.contrib.auth.models import User
+
+from boards.models import Board
+
+
+class AddNewTopicTest(TestCase):
+    def setUp(self):
+        self.board_obj = Board.objects.create(
+            name="board", slug="boarddslug", description="1"
+        )
+        self.user = User.objects.create_user(
+            username="sourabh", email="so@dj.com", password="Somepassword"
+        )
+
+    def test_should_render_error_on_no_form_data(self):
+        url = reverse("new_topic", kwargs={"slug": self.board_obj.slug})
+        data = {}
+        self.client.force_login(self.user)
+        response = self.client.post(url, data)
```

Comment:
Import the form and test it directly. No need to go through client for checking form errors.